tectonic/tectonic.h

```diff
@@ -37,7 +37,12 @@
 
 /* basic types */
 
+#if defined(_MSC_VER)
+// On MSVC it’s already defined if rpcndr.h has been loaded (which it has for some of our files), as this:
+typedef unsigned char boolean;
```

Comment:
The `boolean` typedef has been [removed](https://github.com/tectonic-typesetting/tectonic/commit/f0ec7c6f994b02274d9ee1cd5a08d9cb10a3dbb5). So I guess this change does not make sense now.